documentation/source/base/design_build.dot

```diff
@@ -0,0 +1,96 @@
+// Useful info
+// This graph is designed to be output as an interactive svg
+// fontname, fontcolor and pencolor were copied from the rtd sphinx theme
+// the 'ee' node is used to force alignment, it is set to be invisible
+// target can be modified as per w3 http specs
+
+digraph {
+    compound=true;
+    node [
+        shape = box,
+        width = 2,
+        fontname="Lato,proxima-nova,Helvetica Neue,Arial,sans-serif",
+        fontsize="12px",
+        fontcolor="#404040",
+        color="#404040"
+    ]
+    graph [
+        splines = false,
+        ranksep = 0.2,
+        fontname="Lato,proxima-nova,Helvetica Neue,Arial,sans-serif",
+        fontsize="12px",
+        fontcolor="#404040",
+        pencolor="#404040"
+    ]
+
+    subgraph cluster_0 {
+        label="Designer";
+        tooltip="Designer";
+        href="../base/design_build.html"
+        target="_parent"
+        fontcolor="#2980b9"
+        { rank=same b c ee[group=g1, style=invis, width=0.01, tooltip="Designer"] }
+        a[
+            group=g1,
+            label="ParameterFrame",
+            tooltip="ParameterFrame",
+            href="../base/parameter.html",
+            target="_parent",
+            fontcolor="#2980b9"
+        ];
+        b[label="Optimisation", tooltip="Optimisation"];
+        c[label="Minimal Problem", tooltip="Minimal Problem"];
```

Comment:
I'm not 100% sold on these boxes in the figure. I was kind of expecting the labels to be classes (as they have the same kind of box as the other classes), and, for me, it's also a little ambiguous what 'Minimal Problem' means in this context. It may make the figure a little cleaner as well, were they to be removed.

What do you think?

Reply:
In my view, I don't think we need anything in there. At the highest level the workflow is `designer -> builder -> reactor`, I'm not sure that we need to go any deeper in the 'Introduction' document.

Reply:
done